atlp-rwanda/elite-bn-be
# On successful registration, users should receive a JWT

#### What does this PR do?

This PR should register a new user.

#### How should this be manually tested?

1. Create a `.env` file and add your credentials following an example from the `.env.example` file
2. In your terminal, run the command `npm run migration`  
3. Run `npm start`
4. After connecting to the database, check the following endpoint in your postman;
- `POST` `/api/v1/user/register`

#### Any background context you want to provide?

1. All JavaScript is written in ES6 and it uses babel to transpire down to ES5.
2. The user should receive a token after a successful registration.

#### What are the relevant pivotal tracker stories?

#182201566
